### PR TITLE
readme improvements

### PR DESCRIPTION
**Pull request by wafflepie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,29 +11,27 @@
 </h1>
 
 <h3 align="center">
-🖍️ 🛡 🚀
+🛠 💪 💉
 </h3>
 
 <h3 align="center">
-Modular Redux is possible.
+Maintaining large Redux applications with ease.
```
Comment (tommmyy) [inline README.md]: it is soooo looooong

File: README.md [hunk 2]
```diff
@@ -60,17 +58,15 @@ See the [CHANGELOG.md](CHANGELOG.md) file.
...
 ## License
 
 All packages are distributed under the MIT license. See the license [here](https://github.com/lundegaard/redux-tools/blob/master/LICENSE).
 
 ## Related projects
 
-- [validarium](https://github.com/lundegaard/validarium) - Validations done right.
-- [lundium](https://github.com/lundegaard/lundium) - Beautiful React component library.
-- [react-union](https://github.com/lundegaard/react-union) - Integrate React apps into various CMSs seamlessly.
+- [validarium](https://github.com/lundegaard/validarium) – Validations done right.
```
Comment (tommmyy) [inline README.md]: 🤔

Comment (wafflepie) [reply README.md]: – is better than - :)

Comment (wafflepie) [reply README.md]: (Option + '-')

File: README.md (hunk 1 above)
Comment (wafflepie) [reply README.md]: i think it describes the purpose of this collection better :)

**Pull request merged**